frontends/web/src/routes/account/summary/balancestable.tsx

```diff
@@ -52,7 +52,7 @@ class BalancesTable extends Component<Props> {
                         </tr>
                     </thead>
                     <tbody>
-                        {accounts.map(account => <BalanceRow name={account.name} balance={account.balance.available} code={account.coinCode}/>)}
+                        {accounts.map(account => <BalanceRow key={account.coinCode} name={account.name} balance={account.balance.available} code={account.coinCode}/>)}
```

Comment:
`coinCode` is not unique, it will be `BTC` for e.g. segwit and native segwit. please check if there i `code` or `accountCode`, which would be sth like `btc-p2wpkh` or similar.

Reply:
Resolved, thanks for #1031